addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/discovery/MaxCubeBridgeDiscovery.java

```diff
@@ -44,12 +44,12 @@
  */
 public class MaxCubeBridgeDiscovery extends AbstractDiscoveryService {
 
-    static final String MAXCUBE_DISCOVER_STRING = "eQ3Max*\0**********I";
+    private static final String MAXCUBE_DISCOVER_STRING = "eQ3Max*\0**********I";
     private final static int SEARCH_TIME = 15;
 
-    private final Logger logger = LoggerFactory.getLogger(MaxCubeBridgeDiscovery.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(MaxCubeBridgeDiscovery.class);
```

Comment:
please revert this, see [C.3](http://docs.openhab.org/developers/development/guidelines.html#c-language-levels-and-libraries).